azure-pipelines.yml

```diff
@@ -0,0 +1,60 @@
+name: das-forecasting-jobs
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+workspace:
+    clean: all
+
+variables:
+  BuildConfiguration: 'Release'
+  Build.MajorNumber: '1'
+  Build.MinorNumber: '0'
+
+pool:
+  name: 'Continuous Integration 02 - SSD - 160ACU'
+
+steps:
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet restore'
+  inputs:
+    command: restore
+    projects: 'src/**/*.csproj'
+    noCache: true
+
+- task: DotNetCoreCLI@2
+  displayName: 'Build'
+  inputs:
+    projects: 'src/**/*.sln'
```

Comment:
On line 26 we are restoring the packages for 'src/**/*.csproj' yet we are building 'src/**/*.sln'. These two values should probably match, one or the other.